vowpalwabbit/global_data.h

```diff
@@ -373,8 +709,17 @@ struct vw
 
   AllReduceType all_reduce_type;
   AllReduce* all_reduce;
-
-  bool chain_hash = false;
+
+  InputConfig ic;
```

Comment:
Yep, as outlined in the description of the PR I didn't bother to name properly bc feel that can be decided once and then replace instead of iterating over and over. Did not know how people would react to the word config, etc